---
BEFORE
<await(Promise.reject(new Error("Something went wrong!")))>
    <@then|testData|>Success!</@then>
    <@catch/>
</await>
AFTER
---